website/static/js/headerLinkCopy.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+
+  // Get all the headers with anchor links. 
+  // The 'click' event worked over 'popstate' because click captures page triggers, as well as back/forward button triggers
+  window.addEventListener("click", copyHeader);
```

Comment:
Seeing one scenario where this doesn't fire which may cause the copy function to not work on initial load. Here's a Loom video showing what I'm seeing: https://www.loom.com/share/8ca6256d333d4e669b10b8f500bbaa63?sid=3a49f567-8b8c-48f0-a5bf-4ec16a2086f2

Rather than only listening on `click`, we can update this part to listen for both `load` and `click`. That way if a url is entered in the browser and the page is loaded, a user can click a header immediately, without having to make a click first.

[This post](https://stackoverflow.com/questions/8796988/binding-multiple-events-to-a-listener-without-jquery) on stackoverflow has some hints on how to do this!

Reply:
ok cool! thank you @JKarlavige ! ok i added another evenlistener line focusing on 'load'. i also think i could've done created a function for this but i couldn't get it working.

Reply:
This works great!! No need to update anything here or do things differently!